src/Curators/Publisher.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace CultuurNet\UDB3\Curators;
+
+use InvalidArgumentException;
+
+class Publisher
```

Comment:
I've also used https://github.com/2dotstwice/value-objects for the value objects in the `udb3-models` and `udb3-models-import` libraries (the latter is now included in this codebase as well). It's pretty bare-bones but does the job so we could fork that to the `cultuurnet` organisation and use that perhaps. I'd also be open to another solution but we should make sure we're not introducing [yet another standard](https://xkcd.com/927/).

You can see an example of an enum class using that library here: https://github.com/cultuurnet/udb3-models/blob/master/src/ValueObject/Calendar/CalendarType.php (Note how you for get the validation and static methods for free using the `__construct` and `__callStatic` methods from the [base class](https://github.com/2dotstwice/value-objects/blob/1.x/src/String/Enum.php).)

It's quite experimental though because it uses traits for the low-level stuff (e.g. `IsString`, `IsNotEmpty`, ...). So I'm definitely not saying we have to use this, just more of an FYI on the current ways of creating value objects. My question is mostly where we go from here.

Reply:
Let's look at it when we refactor this out because of the config approach.

Reply:
Alright, it might not be relevant anymore then indeed but it's still something I'd like to look into down the road. Most value objects currently use the base classes from that unmaintained library or the traits from the 2dotstwice library, so I'd like to consolidate those in the long run into something better maintainable.